cdap-ui/app/directives/widget-container/widget-wrangler-directives/widget-wrangler-directives.html

```diff
@@ -14,14 +14,15 @@
   the License.
 -->
 <div class="clearfix">
-  <textarea
-    class="form-control"
-    ng-trim="false"
+  <div
+    my-ace-editor
     ng-model="model"
+    data-config="config"
+    mode="plain_text"
     rows="{{rows}}"
     placeholder="{{placeholder}}"
   >
-  </textarea>
+  </div>
```

Comment:
I think you may need to add the disabled flag in here.. `textarea` could be disabled through our catch all `field` disabled, but ace-editor won't honor that. Make sure to test the behavior on detail view